### ISSUE TITLE
had a problem deploying to PR staging Error

### ISSUE THREAD

**Issue by Foxhunt** [zpnk/stage-ci#35]
Hi, when opening a PR i get this Error message

it links me to [repro-name]-[branch-name].now.sh but ends in a 404.
no now deployment was created on my account

did i mess up my stage-ci setup?

Edit:
found this in the logs of my stage-ci deployment

```
Error: /bin/sh: 1: /stage-ci/node_modules/now/build/bin/now: not found

    at Socket.nowProc.stderr.on (/stage-ci/src/core.js:34:49)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:561:20)
```

**Comment by Foxhunt** [comment]
Just checked the other issues and this seems to be a duplicate of #34.

**Comment by paulirish** [comment]
This looks to be different, actually. 

```
Error: /bin/sh: 1: /stage-ci/node_modules/now/build/bin/now: not found
```
vs
```
Error: /bin/sh: 1:
```

**Comment by paulirish** [comment]
Yah so the bug here seems to be that your dependency isn't found. So i guess it wasn't installed? 

Or this line is failing for whatever reason: `const nowBin = path.resolve('./node_modules/now/build/bin/now');`

**Comment by Foxhunt** [comment]
I redeployed the other day in case something went wrong with my stage-ci deployment.

I can find find both Errors in my logs now.

```
Error: /bin/sh: 1: 
    at Socket.nowProc.stderr.on (/stage-ci/src/core.js:34:49)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:561:20)

Setting GitHub status to "error"...
Done!
```
and 

```
Error: /bin/sh: 1: /stage-ci/node_modules/now/build/bin/now: not found

    at Socket.nowProc.stderr.on (/stage-ci/src/core.js:34:49)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at Socket.Readable.push (_stream_readable.js:136:10)
    at Pipe.onread (net.js:561:20)

Setting GitHub status to "error"...
Done!
```

The Errors occur after checking out the branch on its last commit.
Maybe i find the time to do some local debugging.

**Comment by paulirish** [comment]
Yup. it's a dupe. :) 
thx man

**Comment by hugomd** [comment]
Duplicate of #34

**Comment by zpnk** [comment]
Fixed in [`v1.3.0`](https://github.com/zpnk/stage-ci/releases/tag/v1.3.0).
